ask.py

```diff
@@ -738,7 +740,7 @@ def startup(file_name):
 		# Stats the local development server.
```

Comment:
I noticed a spelling mistake made by me, but maybe you can fix it?

Reply:
on it!